### ISSUE TITLE
[mingw64] [libgme] error with hash

### ISSUE THREAD

**Issue by Jolar** [m-ab-s/media-autobuild_suite#2326]
I had this exact error maybe a month ago when trying to run mabs, but didn't care about looking into it then, and assumed it would be updated later. Maybe the download of game-music-emu-0.6.3 fails? It says "Downloaded" but I can't find the archive anywhere. I also can't figure out where the script is supposed to download the source from, but manually downloaded game-music-emu-0.6.3.tar.xz from ShiftMediaProject/game-music-emu, but got a hash error. I guess much like the output below and in logs.zip, but what you see here is from a completely fresh run. It could be that my error happened because I put the archive in the wrong place. Where should it be? I tried unpacking the archive into 

edit: I just checked that the game-music-emu-0.6.3.tar.xz file I download manually have have the hash aba34e53... as stated down below, so maybe something have changed since this part of the mabs script was made? I tried to unpack the archive in mabs/build/game-music-emu-0.6.3 but the script stopped in this same part, I don't remember why.

``` bash
libgme 0.6.3  ............................................ [Not installed]
┌ game-music-emu-0.6.3  ..................................... [Downloaded]
├ Hash mismatch, file may be broken: aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d != 14f278e4c565221b0822261e024b9282042903df631863b50e3b27211b1425b7 || d45512848bbbba842312ca2db6e55989
├ Running extract...
├ Running cmake...
Likely error (tail of the failed operation logfile):
LDFLAGS: -pipe -static-libgcc -static-libstdc++
cmake .. -G Ninja -DBUILD_SHARED_LIBS=off -DCMAKE_TOOLCHAIN_FILE=/local64/etc/toolchain.cmake -DCMAKE_INSTALL_PREFIX=/local64 -DUNIX=on -DCMAKE_BUILD_TYPE=Release -DENABLE_UBSAN=OFF
CMake Warning:
  Ignoring extra path from command line:

   ".."

CMake Error: The source directory "C:/mabs/build/game-music-emu-0.6.3" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
cmake failed. Check C:/mabs/build/game-music-emu-0.6.3/build-64bit/ab-suite.cmake.log
This is required for other packages, so this script will exit.
```

Logs: https://0x0.st/oI29.zip

**Comment by 1480c1** [comment]
it passed locally, can you try deleting the stuff related to libgme again and rerun?

btw, the url the suite uses is <https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.xz>, not one provided by the ShiftMediaProject (I believe that have forks of multiple projects and add their own modifications for Visual Studio), so the one from there will probably won't match the one from upstream

**Comment by 1480c1** [comment]
I'm probably going to say this is either solved or invalid or something else, if it's still an issue, you can comment, and I'll reopen and re-investigate

**Comment by Jolar** [comment]
Sorry, I haven't been able to respond before. When trying to download the game-music tarball manually from your link on Friday last week, I discovered that it was blocked by the company network. (Crawling slow internet at home.) The download was redirected to bbuseruploads.s3.amazonaws.com which was blocked. Opening bitbucket.org/mpyne/game-music-emu/downloads/ and downloading from there worked though. I put the file in mabs/build, and the script compiled it. (Side note: the direct download worked today.) 

Is it possible to see somewhere where sources are downloaded from? I searched around for quite some time but couldn't find anything.
